docs/docs/platforms/OPENSHIFT.md

```diff
@@ -35,11 +41,11 @@ It depends on the storage backend though.
 
 As an alternative, (if letting containers run as pre-defined users is not possible), set `product_name.securityContextEnabled` to `false`, for example, `confluence.securityContextEnabled: false`
 As a result the container will start as a user with an OpenShift generated ID.
-Typically, NFS permission fixer job isn't required when no security context is set.
+You will also need to disable NFS permission fixer init container as it start as root. Set `volumes.sharedHome.nfsPermissionFixer.enabled=false` to false.
```

Comment:
Minor rephrasing:
"You will also need to disable the NFS permission fixer init container as it starts as root. To do this set `volumes.sharedHome.nfsPermissionFixer.enabled=false` to false."

Reply:
Done